### DISCUSSION TITLE
There is a way to set an auto-update as you release a new fix and version often?

### DISCUSSION THREAD

**Discussion by PeopleInside** [the-djmaze/snappymail#681] [Q&A] [answered]
Hi, will be a way not to much hard to have an auto-update as you release frequent fix and version? :)
Thanks

Umh auto update should also sometimes remove old versions.

**Comment by MI-KY** [comment]
Hi,
I definitely see your point, but isn't a auto update a bit risky 🙂?

**Reply by PeopleInside** [reply]
I am asking the same bug I see when there is a BUG and I update manually the BUG is affecting the webmail as I think will affect with an automatic update. The solution is wait a new update or force manually to use the previous version. I am asking if with the automatic update will be not the same with the difference that if I am not at the PC for manually update to the new version who fix, all will be done by the automatic system. Need also delete some old version automatically when the new version is placed. What I done manually will be nice if can be done automatically :)

**Reply by MI-KY** [reply]
Hi,

yes, your right that if you could automate this it would be better :).
Personally I wouldn't use this because at the moment SnappyMail makes huge and fast steps and sometimes afterwards some new bugs are found. Also i prefere to test new versions before i go to use them on a production system...

It's a decision @the-djmaze has to make :).

**Comment by the-djmaze** [comment, marked as answer]
I hate "auto update" :wink: 
Somehow i always end up with broken websites (wordpress, cs cart, magento, joomla, etc. etc. etc.).

So if SnappyMail has auto-update, you probably also end up with a broken webmail.

Yes, the development is fast, yes there are new bugs, yes eventually it will be stable and has less impact.
But man, i just gave myself a task to "fix" RainLoop and it ended up with "much more todo" hahaha

**Reply by MI-KY** [reply]
Hope you didn't understood my previous text as offense, your really doing a great job! :-).
I just wanted do say the same as you said - as new releases can contian errors, there is always a possibility something will block access to the webmail :-)

**Reply by PeopleInside** [reply]
@the-djmaze understood thanks for your job to maintain active this project.
So for now need update manually and maybe also for the future.
I hope this project will still exist also because seems I found you by searching on the web so your project is important.
Thank you :)
